src/app/_components/account-overlay/account-overlay.component.html

```diff
@@ -41,3 +41,12 @@
       <span i18n="@@account.create_account">Créer un compte</span>
   </button>
 </ng-container>
+
+<button *ngIf="!(isLoggedIn$|async) && !localAuthEnabled && oidcEnabled" mat-button color="primary" (click)="oidcAuthClick()" >
+  <mat-icon>input</mat-icon>
+  <span i18n="@@login.login_oidc">OIDC S'authentifier</span>
```

Comment:
Because this is a new UI element, it's key text need not be french.
How about
"Login with OIDC"
?